pycbc/events/events.py

```diff
@@ -193,6 +193,20 @@ def newsnr(snr, reduced_x2, q=6., n=2.):
         return newsnr
     else:
         return newsnr[0]
+       
+def bluesnr(snr, bchisq, lchisq):
```

Comment:
It's a test combined single detector statistic. I have not idea what to call it. Come up with something.